packages/shared-ui/src/components/canvas/context/FlowGraphContext.tsx

```
@@ -1062,13 +1013,22 @@ export function FlowGraphProvider({ children }: IFlowGraphProviderProps): ReactE
 			setNodes(updated as FlowNode[]);
 			// Recount after update
 			unconfigured = updated.filter((n) => (n.data as any)?.formDataValid === false).length;
+		} else {
 		}
```

ESLint

[error] Empty block statement.

(no-empty)